wgpu-core/src/device/trace.rs

```diff
@@ -14,13 +14,13 @@ type FileName = String;
 pub const FILE_NAME: &str = "trace.ron";
 
 #[cfg(feature = "trace")]
-pub(crate) fn new_render_bundle_encoder_descriptor(
-    label: super::Label,
-    context: &super::RenderPassContext,
-) -> wgt::RenderBundleEncoderDescriptor {
-    wgt::RenderBundleEncoderDescriptor {
-        label: Some(super::own_label(&label).into()),
-        color_formats: context.attachments.colors.to_vec().into(),
+pub(crate) fn new_render_bundle_encoder_descriptor<'a>(
+    label: Option<&'a str>,
+    context: &'a super::RenderPassContext,
+) -> crate::command::RenderBundleEncoderDescriptor<'a> {
+    crate::command::RenderBundleEncoderDescriptor {
+        label: label.map(Cow::Borrowed),
```

Comment:
It might be more consistent to import `Borrowed` directly everywhere.

Reply:
I don't think `wgpu` crate imports `Borrowed` directly anywhere

Reply:
Ah sorry, could have sworn I did the same as in wgpu-rs there.